### ISSUE TITLE
Python Scraper for BS, only show one link

### ISSUE THREAD

**Issue by zdavatz** [openZH/covid_19#262]
Please only show one link in the Python Scraper for BS, only show the link where the number comes from. All other lines also only have one link per source.
```
https://www.gd.bs.ch/, https://www.gd.bs.ch//nm/2020-tagesbulletin-coronavirus-534-bestaetigte-faelle-im-kanton-basel-stadt-gd.html
```
cc @baryluk

**Comment by metaodi** [comment]
There are actually two downloads for the BS scraper and they are delimited with a comma.

I'm not sure which download would be considered the correct one.

This is possibly related to #237, to have a more structured output from the parser.

**Comment by zdavatz** [comment]
The first one `https://www.gd.bs.ch/` just gives an overview. That one is not needed. The second one is the exact one `https://www.gd.bs.ch//nm/2020-tagesbulletin-coronavirus-534-bestaetigte-faelle-im-kanton-basel-stadt-gd.html`

**Comment by baryluk** [comment]
Just use the last one. This is intentional.

**Comment by zdavatz** [comment]
cc @baryluk Would be cleaner if the output would always be formatted the same or at least using CSV standard. It is easier to parse that way. Thank you for your efforts!

**Comment by jb3-2** [comment]
There are sometimes mutliple sources because in the beginning of the pandemy multiple pages were issued per day, e.g. one link with the number of deceased, one with the number of covid-19 cases. Initially we only had one line per day so I added all sources to one day. Should we create multiple lines per day to keep all source links in the dataset?

**Comment by zdavatz** [comment]
> There are sometimes mutliple sources because in the beginning of the pandemy multiple pages were issued per day, e.g. one link with the number of deceased, one with the number of covid-19 cases. Initially we only had one line per day so I added all sources to one day. Should we create multiple lines per day to keep all source links in the dataset?

Would be great if we could stick to the CSV standard and always use `;` as delimiter.

**Comment by baryluk** [comment]
Also, I am working on extending a TI scraper from scrape two different sources. One for confirmed cases and deaths, and another for extra information (hospitalization, ICU, vents), but it might or might not be available on any given day. So there will be two URLs most likely anyway.

**Comment by zdavatz** [comment]
> Also, I am working on extending a TI scraper from scrape two different sources. One for confirmed cases and deaths, and another for extra information (hospitalization, ICU, vents), but it might or might not be available on any given day. So there will be two URLs most likely anyway.

Great. But then please format the output in the standard CSV way and use a delimiter like `;` or `|`.

**Comment by jb3-2** [comment]
@zdavatz Yes of course, that‘s what I did, but it got lost when multiple columns got merged into one. 
@baryluk Great, I‘d love to not lose any source URLs. 
Great job to all of you, thank you so much for your work!

**Comment by metaodi** [comment]
Fixed with #296
